Create chapter_0 Readme.md

timsjpark opened the pull request:
[#104389596]

jasonnoble commented on chapter_0/Readme.md:
```diff
@@ -0,0 +1,28 @@
+
+**1) Provide a link in your Readme.md that links to the documentation for the following classes:**
+
+1. Array - http://ruby-doc.org/core-1.9.3/Array.html
```
Should probably link to the Ruby 2.2.3 version of the docs, since that's what we're using.

jasonnoble commented on chapter_0/Readme.md:
```diff
@@ -0,0 +1,28 @@
+**1) Provide a link in your Readme.md that links to the documentation for the following classes:**
+
+1. Array - http://ruby-doc.org/core-1.9.3/Array.html
+2. Hash - http://ruby-doc.org/core-2.2.2/Hash.html
```
2.2.3 docs...

jasonnoble commented on chapter_0/Readme.md:
```diff
@@ -0,0 +1,28 @@
+
+**2) Find two other relative ruby documentation links and add those to the Readme.md as well.**
+
+1. Time - http://ruby-doc.org/core-2.2.0/Time.html
```
2.2.3

jasonnoble commented on chapter_0/Readme.md:
```diff
@@ -0,0 +1,28 @@
+**2) Find two other relative ruby documentation links and add those to the Readme.md as well.**
+
+1. Time - http://ruby-doc.org/core-2.2.0/Time.html
+2. String - http://ruby-doc.org/core-2.2.0/String.html
```
2.2.3

jasonnoble commented:
Couple small changes, need to add me to the repo...

jasonnoble commented:
Needs rebasing because #1 was merged...

jasonnoble commented:
You link to two different stories???  [#104389596] and [#104391102]

#attention2detail

jasonnoble commented:
![rebase](https://cloud.githubusercontent.com/assets/22501/10214203/1ffe7bb2-67d3-11e5-9c2b-0b862617b581.jpg)

timsjpark commented:
This branch was named improperly. The reason I had two different tracker IDs in my commit messages was because of a confusion with a different tracker story. I have renamed my local branch to 104389596_chapter_0_master and tracked it to an appropriate branch on origin. Then I deleted this branch.

timsjpark commented:
I guess I need to sleep more and pay more 'Attention To Detail'
